pkg/health/clusterStatsHandler.go

```diff
@@ -175,8 +175,6 @@ func convertDataToSlice(allIndexStats utils.AllIndexesStats, volumeField, countF
 		nextVal[index][columnCountField] = humanize.Comma(int64(indexStats.NumColumns))
 		nextVal[index][earliestEpochField] = time.Unix(int64(indexStats.EarliestTimestamp/1000), 0).UTC().Format("2006-01-02 15:04:05") + " UTC"
 		nextVal[index][latestEpochField] = time.Unix(int64(indexStats.LatestTimestamp/1000), 0).UTC().Format("2006-01-02 15:04:05") + " UTC"
-		nextVal[index][recordCountField] = humanize.Comma(int64(indexStats.NumRecords))
-		nextVal[index][bytesReceivedField] = float64(indexStats.NumBytesIngested)
```

Comment:
Just curious: why are you removing this?

Reply:
There was some duplicate information such as `bytes received` and `incoming volume`, as well as `num of records` and `event count`.